packages/cml/src/replaceCmlPlaceholders/replaceCmlPlaceholders.ts

```diff
@@ -78,6 +78,16 @@ const replacePropValues = (mdx: string, values: Record<string, string> = {}): st
       case 'number':
         searchValue = match;
         newValue = `{${propValue}}`;
+        break;
+      case 'json':
+        searchValue = match;
+
+        if (propValue && typeof propValue === 'object') {
+          newValue = `{${JSON.stringify(propValue)}}`;
+        } else {
+          newValue = `{${propValue}}`;
```

Comment:
can we just do?
```suggestion
          newValue = propValue;
```

Reply:
Yeah. If that works. 
But I thought it should be `<Component prop={{json: 'data'}} />`. That's why I have set it like that.

Reply:
Yeah. If that works. 
But I thought it should have the extra curly braces like `<Component prop={{json: 'data'}} />`. That's why I have added it.

Would it work if we pass it like this `prop={json: 'data'}`?
If so I can update the test cases.

<img width="523" alt="Screen Shot 2023-07-06 at 16 45 52" src="https://github.com/cmpsr/composer/assets/1816129/b582579d-a28a-4a0b-92b1-bc415bdb58d5">

Reply:
tested and it works as it is